logging/logs.rb

```diff
@@ -0,0 +1,19 @@
+# Types of log statements:
+#   log.debug "a very nice little debug message"
+#   log.info "things are operating nominally"
+#   log.warn "this is your last warning"
+#   log.error StandardError.new("something went horribly wrong")
+#   log.fatal "I Die!"
+
+require 'logging'
+require 'environment.rb'
+@file = ENV['logging/log_file.log']
```

Comment:
So this would require us to have an environment variables called `logging/log_file.log` which would be set to the path of our log file. The more standard way to do this is to name the variable `LOG_FILE_PATH` which we would set in `dockerfiles/app.env.dist` to `LOG_FILE_PATH=/path/to/log_file.log`.